backend/internal/middleware/auth.go

```diff
@@ -14,40 +15,39 @@ type AuthenticatePayload struct {
 
 func Authenticate() gin.HandlerFunc {
 	return func(ctx *gin.Context) {
-		var sessionToken string
-
-		cookie, err := ctx.Request.Cookie("sessionToken")
-		if err != nil {
-			// If no cookie, check the body for the session token
-			sessionToken = ctx.GetHeader("Authorization")
-
-			if sessionToken == "" {
-				// There was no cookie, nor sessionToken in the body
-				fmt.Println("[-] No cookie, no sessionToken, or invalid request.")
-				fmt.Println(err)
-				jsonUnauthorized(ctx)
-				return
-			}
-		} else {
-			sessionToken = cookie.Value
-		}
-
-		payload, err := security.VerifyJWT(sessionToken)
+		var sessionToken string = ctx.GetHeader("Authorization")
+
+		// cookie, err := ctx.Request.Cookie("sessionToken")
+		// if err != nil {
+		// 	// If no cookie, check the body for the session token
+
+		// 	if sessionToken == "" {
+		// 		// There was no cookie, nor sessionToken in the body
+		// 		fmt.Println("[-] No cookie, no sessionToken, or invalid request.")
+		// 		fmt.Println(err)
+		// 		jsonUnauthorized(ctx)
+		// 		return
+		// 	}
+		// } else {
+		// 	sessionToken = cookie.Value
+		// }
```

Comment:
Why?

Reply:
LOL forgot about this... when I commented this piece of code all my problems were solved... now it's just a case of uncommenting and figuring out how to make it so that it's sole existence doesn't break the get current user functionality